Stormshield/_meta/manifest.yml

```diff
@@ -0,0 +1,6 @@
+uuid: 758103e9-eb62-4b0d-8c71-0ff9c60ca677
+name: Stormshield
+slug: stormshield
+description: When you choose Stormshield to protect your critical and operational infrastructures, you’re building a relationship of trust with men and women who are passionate about what they do. Stormshield the European cybersecurity choice.
```

Comment:
I wonder if we should not moderate the enthusiasm of the description? Looks like a commercial advertising.